ocrs-cli/src/main.rs

```diff
@@ -105,6 +105,9 @@ struct Args {
     /// Extract each text line found and save as a PNG image.
     text_line_images: bool,
 
+    /// Recognize text using white listed character only
+    white_list: Option<String>,
```

Comment:
Can you rename this to `allowed_characters`, and the same for other identifiers and CLI arguments. I think this will be more immediately obvious. I would also add a note that this should be a subset of the characters in `alphabet`.

Reply:
Ok 👍.